### ISSUE TITLE
Renaming an extension

### ISSUE THREAD

**Issue by twiro** [symphonists/symphony-extensions-network#2]
Supposed I have an extension named "Slider" and would like to rename it to "Slider Field".
Would that be possible in the symphony extension eco-system or would I face unsolvable problems?
1. On **github** itself it wouldn't be a problem.
2. How would it work on **symphonyextensions.com**? Would it be possible? Would I have to post an issue here and request renaming or could I do it myself?
3. Regarding **Symphony CMS** itself. Would it be possible to change the name of an extension while updating or would this be impossible due to the way Symphony uses names to identify extensions? If so would this mean, that a renamed extension would only be possible to include in Symphony as a **new** extension?

Would be great to clarify this and would be some useful information to include in a kinde of "FAQ" in the readme of this repo.

@nitriques - I'd like to label this issue with `FAQ`, but it looks like I can't create new labels. Could you take care of this?

**Comment by nitriques** [comment]
> I'd like to label this issue with FAQ, but it looks like I can't create new labels. 

And now ? Can you ?
1. Effectively, it's not a problem...
2. Right now, this is manual, but it could be automated since github supports redirects.
3. Can you try it ? I think it is possible!

**Comment by twiro** [comment]
> > I'd like to label this issue with FAQ, but it looks like I can't create new labels.
> 
> And now ? Can you ?

Yes :) Thank you! I started with a [list of tags](https://github.com/symphonists/symphony-extensions-network/labels) that might serve well as topics for the readme too.

> Can you try it ? I think it is possible!

Wouldn't know where to start... If I get it right the updater would have to change the extensions folder name and the name of the class in the extension driver - do you think this would be possible and do you know of an extension where this has been already done and that might serve as an example?

If not - don't care... it's not that important!

**Comment by nitriques** [comment]
Ah I thought I meant just rename the extension on the site...

Best bet would be to create a new repo I think!
